expand on why we use Metup and why events must be registered there (#124)
rladies/rladiesguide: content/coordination/meetup/index.en.md

Ideally, this will become a team of at least 2 people, to split the workload
and guarantee requests are addressed any time of the year.

## Why do we use Meetup?

> Meetup might not be well-known to you or your region, but it has been a very 

Reviewer: Please add line breaks after each sentence. :pray:

Reviewer 2: on my end it looks like there are line breaks??

important platform for many to establish a community. Members can "join" your 
chapter's page and then subscribe to all your events. This makes it easier 
for people to never miss an event announcement. 

While you may prefer using other platforms that reach more people in your area,

> we will still require you to register your events on Meetup. Meetup provides the 

Reviewer 2: ```suggestion
we still require you to register your events on Meetup. Meetup provides the 
```

most straight forward way for the Global team to obtain information about chapter

> activity and members. Despite these numbers likely not 100% reflecting chapter 
> memberships, at minimum we can have a singular platform where events are registered.

Reviewer 2: ```suggestion
activity and members.
```

Reviewer 2: didn't think this added much for the reader

Reviewer 3: Organizers from one chapter that is not using Meetup were worried that their membership numbers are not reflected there. So, I think this is an answer for those who are worried that their activity is being judged by the number of members on that platform

> The regitering of events on Meetup, makes it possible for the Global Team to 

Reviewer: ```suggestion
The registering of events on Meetup makes it possible for the Global Team to 
```

Reviewer: this could be integrated with the second sentence of this paragraph "Meetup provides the..."

create chapter activity reports to our funders, which increases the likelihood
that funders will continue providing funds for R-Ladies. These funds we use
to establish global infrastructure, like the Zoom account, E-Mails, Website,
Calendly for bookings etc. 

> 

Reviewer 4: ```suggestion
A Meetup account is also a requirement for the chapter to be eligible for the [RConsortium RUGs grants](https://www.r-consortium.org/all-projects/r-user-group-support-program). 
```

> The registering of events on Meetup makes it possible for the Global Team to 
> create chapter activity reports to our funders, which increases the likelihood
> that funders will continue providing funds for R-Ladies. These funds we use
> to establish global infrastructure, like the Zoom account, E-Mails, Website,
> Calendly for bookings etc. 
> 
> A Meetup account is also a requirement for the chapter to be eligible for the [RConsortium RUGs grants](https://www.r-consortium.org/all-projects/r-user-group-support-program). 

Reviewer 2: ```suggestion
Registering events on Meetup enables the Global Team to create chapter activity reports
required for our funders. These reports are then used to justify a continuation (or modification, 
if needed) of our funding agreement. These funds are used to pay for for global operations,
 including Meetup, Zoom, E-Mails, Website hosting, Calendly for bookings, and more.

A Meetup account is also a requirement for the chapter to be eligible for the [RConsortium RUGs grants](https://www.r-consortium.org/all-projects/r-user-group-support-program). 

Thank you for helping the Global Team track chapter activity so that we may continue
to secure the funding that serves this community.
```
